### PR TITLE
UI creationOOP school library: add basic UI

### PR DESCRIPTION
**Pull request by khanh-devos**
- [x] Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
- [x] In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        student_or_teacher
+      when 4
+        create_book
+      when 5
+        create_rent
+      when 6
+        list_rentals
+      else
+        break
+      end
```
Comment (Gambit142) [inline app.rb]: - [ ] Great work. Your console app works well 👏. There is a little detail, you missed out. Based on this [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), when presented with the list of options and the user types the option `7`, it should display **_'Thank you for using this app!'_** and exit the application. Kindly implement this change. 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @khanh-devos 👋, 

Good job so far!

## Highlights
- Setup linters correctly ✔️ 
- Created app entry point ✔️ 
- Console's behaviour is similar to that of the video ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app.rb (hunk 1 above)
Comment (khanh-devos) [reply app.rb]: Thank you, @Gambit142 
Got it

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @khanh-devos   

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [x] Good project organization ✔️
- [x] Nice use of Ruby features ✔️ 
- [x] Kudus on meeting the requirements suggested  ✔️

### Confirmed view:
image


Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by khanh-devos** [comment]
Thank and have a good weekend, @Shedrack-Sunday

**Pull request merged**